### PR TITLE
FAI-596 UI test: Outcome - simple value

### PR DESCRIPTION
**Pull request by sjamboro**
[FAI-596: UI test: Outcome - simple value](https://issues.redhat.com/browse/FAI-596)

Goal: assert that simple outcome value is visible on the page and accesible via OUIA

Perform tests: ```yarn run test:it``` in the `ui-packages/packages/trusty` folder

### CODE REVIEW

File: ui-packages/packages/trusty/it-tests/integration/Outcomes.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          'has.text',
+          'Null'
+        );
+        cy.ouiaId('view-detail', 'PF4/Button').should('be.visible');
+      });
+      cy.ouiaId('Risk Score', 'PF4/Card').within(() => {
+        cy.ouiaId('card-title', 'title').should('has.text', 'Risk Score');
+        cy.ouiaId('Risk Score', 'simple-property-value').should(
+          'has.text',
+          '21.7031851958099'
```
Comment (manstis) [inline ui-packages/packages/trusty/it-tests/integration/Outcomes.ts:43]: I'd be a little worried that this is really precise. 

Does this test use mocked data to guarantee this value can never vary?

Comment (sjamboro) [reply ui-packages/packages/trusty/it-tests/integration/Outcomes.ts:43]: @manstis yes, it is performed with mocked data (mention in JIRA - https://github.com/kiegroup/kogito-apps/blob/main/ui-packages/packages/trusty/api-mock/mocks/outcomes.js#L23). Unfortunately it is the only simple value in mocked data. Is necessary to show 13 decimal places?

**Comment by sjamboro** [comment]
Jenkins retest this

**Comment by sjamboro** [comment]
Jenkins retest this

**Review by jstastny-cz — APPROVED** [review]
LGTM.

**Review by manstis — APPROVED** [review]

**Comment by manstis** [comment]
Jenkins retest this

**Pull request merged**